packers/jws.go

```diff
@@ -74,10 +74,9 @@ type JWSPacker struct {
 
 // SigningParams packer parameters for jws generation
 type SigningParams struct {
-	SenderDIDstr string
```

Comment:
In some cases we may not have the complete did document

Reply:
???

Reply:
@vmidyllic I remember we discussed that users can have only DID not the whole DID document. But now we removed this possibility.